### DISCUSSION TITLE
p/4580ae0544204ef2a30f3ce496f64d7b/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#55] [Announcements]
# p/4580ae0544204ef2a30f3ce496f64d7b/

又过了两周，西藏之旅结束了。这次西藏之旅非常不完美，但可以说仍然是一种独特的体验。之前上过 4500+ 的可可西里，所以对于高原反应没怎么重视，结果后面一周在林芝一直处于感冒之中。  拉萨在经过人生中最长的火车后（67 小时），终于到达了青藏高原 – 拉萨。路上的风景确实不错，但火车实在太久，还是建议先来到西宁然后坐火车去拉萨，可能会更轻松一些。 这儿的云看起来都很低，从山后翻涌而来。  路上经过

https://blog.rxliuli.com/p/4580ae0544204ef2a30f3ce496f64d7b/

**Comment by miaopass-future** [comment]
哇哇，好羡慕满世界跑，然后还不用承受工作的重压。
你是一个人旅行的嘛

**Reply by rxliuli** [reply]
嗯，一个人很自由
